exercises/numbers/isZero/isZero.js

```diff
@@ -5,6 +5,8 @@
  * @returns {boolean} True if num is 0 and false otherwise
  */
 function isZero(num) {
+
+  //return the variable value //
```

Comment:
We're returning the value of the expression `num === 0`.

If we call, say, `isZero(1234)` then inside the function:

- The value of `num` will be `1234`
- `num === 0` will evaluate to `false` (since `1234` does not equal `0`)

We return what `num === 0` evaluates to, not the value of a specific variable.